card-1/css/_global.scss

```
@@ -0,0 +1,27 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@400;700&display=swap');
+
+*{
```

```suggestion
* {
```
i u ovom fajlu ima slicnih gresaka